Testskript.R

```diff
@@ -0,0 +1 @@
+# Hallo Jule, das ist unser Testskript :-)
```

Comment:
Awesome, you did amazing!
But please add one more smiley! ;)

Reply:
Good idea, thanks for your input!